delegation/test/cardano-sl-delegation-test.cabal

```diff
@@ -0,0 +1,33 @@
+name:                cardano-sl-delegation-test
+version:             1.1.0
```

Comment:
aren't we at 1.2.0 ?

Reply:
Supposed to change to 1.3.0. Will get that done.

Reply:
Looks like I changed it but didn't push the commit... my bad.